doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -381,6 +422,51 @@ There are three situations:
     Check (b 0).
     Unset Printing Coercions.
 
+.. _example-reverse-coercion:
+
+.. example:: Reverse coercions.
+
+  .. coqtop:: in
+
+    Structure S := {
+      ssort :> Type;
+      sstuff : ssort;
+    }.
+    Definition test (s : S) := sstuff s.
+    Canonical Structure S_nat := {| ssort := nat; sstuff := 0; |}.
+
+  .. coqtop:: all
+
+    Check test (nat : Type).
+
+.. _example-reverse-coercion-attribute:
+
+.. example:: Reverse coercions (reverse attribute).
+
```

Comment:
```suggestion
Notice there is no `:>` on `ssort'` and the added :cmd:`Coercion` compared to the previous example
```

Would be so nice if we could boldface parts of examples.  You can put comments in `.. coqtop::` but only between Coq sentences.